tests/activity.go

```diff
@@ -1114,7 +1114,8 @@ func (s *FunctionalSuite) TestActivityHeartBeat_RecordIdentity() {
 
 func (s *FunctionalSuite) TestActivityTaskCompleteForceCompletion() {
 	sdkClient, err := sdkclient.Dial(sdkclient.Options{
-		HostPort:  s.testCluster.GetHost().FrontendGRPCAddress(),
+		// TODO: do load-balancing
+		HostPort:  s.testCluster.GetHost().FrontendGRPCAddresses()[0],
```

Comment:
Can you keep FrontendGRPCAddres?
`func (h * XXX) FrontendGRPCAddress() {
    return h.FrontendGRPCAddresses()[0]
}`

it will be cleaner that way.

Reply:
True. I can also do a followup PR to make the SDK load-balance